django_project/certification/models/certifying_organisation.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+"""
+Certifying organisation model definitions for certification apps
+"""
+
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils.text import slugify
+from core.settings.contrib import STOP_WORDS
+from unidecode import unidecode
+from django.contrib.auth.models import User
+from django_countries.fields import CountryField
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SlugifyingMixin(object):
```

Comment:
No, although other codes uses slugifying too, they don't specify it as one specific class that can be used in other models.